FBSimulatorControl/Configuration/FBSimulatorControlConfiguration.m

```diff
@@ -48,7 +55,7 @@ - (instancetype)copyWithZone:(NSZone *)zone
 
 - (NSUInteger)hash
 {
-  return self.simulatorApplication.hash | self.namePrefix.hash | self.bucketID | self.options;
+  return self.simulatorApplication.hash | self.deviceSetPath.hash | self.namePrefix.hash | self.bucketID | self.options;
```

Comment:
`xor` makes a lot more sense I think. Will do this everywhere for all the hashes we have.